ddtrace/logger.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from logging.handlers import RotatingFileHandler
+
+from ddtrace.internal.utils.formats import asbool
+from ddtrace.utils.formats import get_env
+
+
+def configure_logger():
+    logging_level = asbool(get_env("trace", "debug", default=False))
+    log_path = get_env("trace", "log", "file", default="default")
+    max_file_bytes = get_env("trace", "log", "file", "size", "bytes", default=15000000)
+    num_backup = 1
+    log_format = "%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] {}- %(message)s"
+
+    ddtrace_logger = logging.getLogger("ddtrace")
+
+    if logging_level:
```

Comment:
Some small suggestions to the logic here.

We should allow people to set `DD_TRACE_LOG_FILE` but not `DD_TRACE_DEBUG`, meaning we could allow someone to redirect the library logs, but we'll configure a sane default level like "warning", and only set the level to "debug" if they set `DD_TRACE_DEBUG`.

What do you think? Would that be valuable?

Reply:
I think that's a fair setting to provide in the case that we're only looking for WARNING or lower level type logs. I tried adding this logic in the latest commit, but I'm not sure what to do about the case where a path hasn't been provided, but a debug setting was configured.